jwst/resample/resample_spec_step.py

```diff
@@ -44,6 +33,13 @@ def process(self, input):
         for reftype in self.reference_file_types:
             ref_filename = self.get_reference_file(input_models[0], reftype)
 
+        if ref_filename != 'N/A':
+            self.log.info('Drizpars reference file: {}'.format(ref_filename))
+            kwargs = self.get_drizpars(ref_filename, input_models)
+        else:
+            # Deal with NIRSpec which currently has no default drizpars reffile
```

Comment:
I assume this is a temporary workaround until someone creates a drizpars reffile for NIRSpec?

Reply:
Yes, it also is a general "if there is no reffile" workaround for any undefined modes that don't have a reffile.  NIRSpec is the one that I ran into that did not have a reffile, but perhaps there are others.  NIRCam, MIRI and NIRISS have reffiles right now.  So perhaps FGS needs one too?  Not sure they'll be resampling FGS science data.